### ISSUE TITLE
API Request: Persistence

### ISSUE THREAD

**Issue by peterjcaulfield** [pmndrs/leva#113]
An easy way to persist values (local storage is fine) and easy way to clear the storage e.g (save/clear buttons).

It would be great to just have this as a boolean config that would appear at top of the pane. If we are specifying the leva pane manually:

```jsx
<LevaPane persistControls />
```

You could even add an autosave the same way:

```jsx
<LevaPane persistControls autoSave />
```

an key/value object config could be used for the global panel.

**Comment by dbismut** [comment]
I agree we need to consider this. I'm just concerned about what happens with multiple stores (maybe that option is reserved for the global store panel) and when you change an input's path.

**Comment by izmaelmag** [comment]
Any updates on this 3 years later?
